mne_bids/tests/test_write.py

```diff
@@ -271,7 +271,7 @@ def test_create_fif(_bids_validate):
     _bids_validate(bids_root)
 
 
-@requires_version('pybv', '0.3')
+@requires_version('pybv', '0.4')
```

Comment:
Why don't we simply remove these? Developers should have pybv installed to run the tests, and we don't want any tests to be silently skipped, no?

Reply:
mmh yes, I don't actually know why we have these in the test scripts.

Reply:
no we don't want to impose to users all dependencies. If they don't need pybv the tests should not break for them.

also for linux packaging eg on debian if we do this it means we need a debian packages for pybv otherwise mne will not pass the tests. Not sure you have the bandwidth to package pybv on debian :)

Reply:
But to run the tests, you also need bids-validator, and this requires node, … And which ordinary user grabs and runs the tests, anyway? The recommended installation is via pip or conda, and the sdist doesn't contain any tests.

Reply:
ahh I remember why we had this! If we run a test with "minimal dependencies", the tests involving optional dependencies need to be skipped.

I think this PR can be merged as is